static/gsApp/views/subscriptionPage/reservedUsageChart.spec.tsx

```diff
@@ -62,6 +65,48 @@ describe('mapStatsToChart', () => {
   });
 });
 
+describe('mapReservedToChart', () => {
+  it('should apply GIGABYTE multiplier for byte categories (attachments)', () => {
```

Comment:
nit: you could just cut down the tests to three for byte categories, duration categories, and count categories (plus the two edges cases for unlimited and null) since there isn't actually any differentiation between individual categories within those subsets